project/app/src/main/AndroidManifest.xml

```diff
@@ -26,8 +26,6 @@
             android:stateNotNeeded="true"
             android:windowSoftInputMode="stateAlwaysHidden" />
         <activity android:name=".activities_trials.ChooseConductActivity" />
-        <activity android:name=".activities_qr.CountQRActivity" />
```

Comment:
Why are these removed?

Reply:
I was originally going to have separate activities for generating QR codes of each trial type and one more activity to display the generated code, but I managed to refactor all of it into one single activity, so I deleted those old ones 👍